src/containers/RecordingView.tsx

```diff
@@ -32,6 +33,8 @@ interface OwnProps {
   recordingId: string
 }
 
+const calculationPool = createCalculationPool({maxSimultaneous: 6})
```

Comment:
We'll need to figure out a more systematic way to assign calculation pools. It's a client-side mechanism for preventing too many simultaneous hither calls to the server.